pkg/namespaces/namespaces.go

```diff
@@ -33,6 +33,35 @@ func (n UsernsMode) Valid() bool {
 	return true
 }
 
+// CgroupMode represents cgroup mode in the container.
+type CgroupMode string
+
+// IsHost indicates whether the container uses the host's cgroup.
+func (n CgroupMode) IsHost() bool {
+	return n == "host"
+}
+
+// IsContainer indicates whether the container uses a new cgroup namespace.
+func (n CgroupMode) IsContainer() bool {
+	return n == "container"
```

Comment:
Shouldn't this `Split()` to get rid of the container ID after `container:`?

Reply:
I really used "container" to mean "new namespace".  This probably clashes with the `container:ID` notation.  What do you suggest for using?

Reply:
The way inspect handles it is generally `container:$ID` refers to a separate container, `host` for the host namespace, `private` for a new CGroup namespace